src/Chapter4.hs

```diff
@@ -611,6 +641,17 @@ Implement the 'Monad' instance for our lists.
   maybe a few) to flatten lists of lists to a single list.
 -}
 
+instance Monad List where
+    (>>=) :: List a -> (a -> List b) -> List b
+    (>>=) aList f =
+      let
+        (+++) (Cons a t) as = Cons a (t +++ as)
+        (+++) Empty as = as
```

Comment:
You can move this `+++` operator to the top level to avoid duplicating it in the `Applicative` instance 🔝